BE/src/main/java/com/codesquad/issue04/service/UserService.java

```diff
@@ -36,20 +36,20 @@ public AllAuthorsResponseDto getAllAuthors() {
             .collect(Collectors.toList());
     }
 
-    public List<Issue> getAllAssignedIssues() {
-
-        String userId = "guswns1659";
-        RealUser realUser = userRepository.findByGithubId(userId)
-            .orElse(NullUser.of());
-
-        return realUser.getAssignedIssues();
+    public AllAssigneeResponseDto getAllAssignee() {
+        return AllAssigneeResponseDto.builder()
+            .allData(findAllAssignee())
+            .build();
     }
 
-    public List<RealUser> findUsersHasIssues() {
+    private List<AssigneeDto> findAllAssignee() {
         List<RealUser> users = userRepository.findAll();
-
-        return users.stream()
+        List<RealUser> allAssignee = users.stream()
             .filter(RealUser::hasAssignedIssues)
             .collect(Collectors.toList());
```

Comment:
```suggestion
return users.stream()
	.filter(RealUser::hasAssignedIssues)
	.collect(collectingAndThen(toList(), AllAssigneeResponseDto::of);
```

현재 스트림을 여러 개 만들고 있는데요. 스트림을 여러 번 생성하면 메모리를 상당히 많이 잡아먹어 성능에 좋지 않습니다. 하나의 스트림만 생성하기를 권합니다. 따라서 AllAssigneeResponseDto를 바로 반환하려고 할 때, 이런 식으로 처리할 수 있습니다. ```collectingAndThen```에 대해 학습해볼 것을 권합니다.
위의 코드는 제가 그냥 생각나는 대로 친 거라 컴파일 에러가 날 가능성이 높으니까 바로 적용하지는 마시구요.